community/becoming-a-maintainer.md

```diff
@@ -1,3 +1,77 @@
+
+## Responsibilities
+
+Maintainers are responsible for ensuring that a Track works well within Exercism, and continues to grow and develop. 
+That specifically means that maintainers:
+- **Set technical direction for a track:** Choosing the right language versions, tooling, etc.
+- **Manage the syllabus of a track:** Designing, choosing and defining the right Concepts and Exercises for a Track, and ensuring they are linked and structured appropriately.
+- **Ensure a traack track work well:** Ensuring the Track is up to date, that its test-runner is fast, etc.
+- **Ensure the track conforms to Exercism's standard:** Exercism has specifications and standards for each language Track to ensure a degree of consistency within the platform. While maintainers have free reign over much of the Track, all Tracks must conform to Exercism's standards.
+
+Maintainers are also responsibile for interacting with our contributor community.
+They review PRs from community, acting as gatekeepers, shephards, and cheerleaders for our community.
+They are responsible for keeping documentation and contribution guidelines up to date, and ensuring that there are relevant open issues for potential contributors to action.
+
+It is essential that Maintainers manifest Exercism's values of empathy and kindness, while maintaining high standards for the Track and the platform.
+
+## Powers
+
+The primary way that Maintainers are able to enact their responsibilities is by having the write access to repositories. 
+Through this Maintainers control what code goes into their repositories.
+
+## Rewards
+
+The main incentives for being an Exercism Maintainer are intrinsic: working on a meaningful project; working with other smart, kind people; developing both technical and soft skills; having a meaningful out-of-work entry on your résumé.
+
+We also provide some extra benefits in:
+- A "Maintainer" badge on profiles and around the website.
+- Blog posts and interviews with maintainers.
+- Exclusive Maintainer swag
+
+## Maintainer requirements 
+
+Maintainers commit themselves to spending an average of ~4 hours per week on maintaining their Track(s).
+
+### Path to becoming a Maintainer
+
+On tracks that are actively maintained, you will be invited to become a Maintainer when you achieve 150 building and/or authoring reputation on that Track within a 6 month period. 
+
+On tracks without maintainers, we have less rigorous restrictions.
+After some small merged contributions, you can open an issue on the Track asking to become a Maintainer.
+
+### Staying a maintainer
+
+Every six months we check to see if our Maintainers are still active. 
+In those checks, anyone who has built 100 Reputation from building/authoring/maintaining track in previous 6 months is invited to continue being a maintainer.
+In general those who have done little or nothing during that window, will be removed as Maintainers.
```

Comment:
Perhaps this should specify that there are exceptions, for example when there's little to do on the track overall.

Reply:
Agreed.

Reply:
```suggestion
In general those who have done little or nothing during that window, will be removed as Maintainers.
This is a guiding principle not an absolute rule, and we take into consideration a person's personal circumstances and the state of Track before confirming these decisions.
```